.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,12 @@
+# Info
+
+<!--
+* Please don't force push to PRs, it removes the history, creates bad notifications, and we will squash and merge in the end.
+* Feel free to ping for a review after a few days (@simondanisch, @ffreyer). We can't guarantee a review in a certain time frame, but we can guarantee to forget about PRs after a while.
+* The easier a PR is to review, the easier we can merge it:
+    * The more tests you add, the easier it is to see that your change works
+    * The clearer the problem being solved the easier.
+-->
```

Comment:
```suggestion
* Allowing write access on the PR lets us merge more easily, since we can make edits to the PR directly and update it if it gets out of sync with `master`.
-->
```

Reply:
I think we do that already automatically? Or is there a way to deny that?
I at least seem to have write access to most PRs.

Reply:
I believe it's a checkbox that has to be actively selected.